script.artwork.downloader/lib/provider/tmdb.py

```diff
@@ -138,16 +141,47 @@ def get_image_list(self, media_id):
                 return image_list
 
 
+#def _search_movie(medianame,year=''):
```

Comment:
I would suggest to remove this? If we ever wanted to know what this method did, we can always use the GIT history?

Reply:
Basically I didn't touch any of this beside the first one you mentioned that remains there due to a small debug that I was try to investigate. As I said I just fix the API Keys issue and add them to the Settings, and fix some Fanart.tv scraper issues.

Reply:
In addition, look about @Lunatixz comment, probably I will return this instead the new method there.